src/document.c

```diff
@@ -411,8 +411,8 @@ FIELD_PREPROCESSOR(fulltextPreprocessor) {
     uint32_t newTokPos;
```

Comment:
This variable is no longer needed; remove please.

Reply:
also, `lastTokPos` should be initialized later on..

Reply:
Done